pandas/core/strings.py

```diff
@@ -484,8 +518,9 @@ def str_get_dummies(arr, sep='|'):
     1  0  0  0
     2  1  0  1
 
-    See also ``pd.get_dummies``.
-
+    See Also
+    --------
+    ``pandas.get_dummies``.
```

Comment:
this actually raises an error during the HTML build for me, I think it needs to be __pandas.get_dummies__ without any quotes / backticks